profiling-ffi/build.rs

```diff
@@ -5,6 +5,7 @@ extern crate build_common;
 use build_common::generate_and_configure_header;
 
 fn main() {
+    println!("cargo:rerun-if-changed=src");
```

Comment:
Do you remember why you added this? Seems like an odd thing to have to add, it should already know to rebuild if one of its rust source files changes, right?